Lib/WebHandler.py

```diff
@@ -1,28 +1,33 @@
-
-from DIRAC import gLogger
-from DIRAC.Core.Security.X509Chain import X509Chain
+from DIRAC import gLogger, S_OK, S_ERROR
 from DIRAC.Core.Security import Properties
-from DIRAC.Core.DISET.ThreadConfig import ThreadConfig
```

Comment:
hm. This is used no? When it is removed? This is used to deal with DISET configuration

Reply:
It's just moved a few lines below.

Reply:
All libraries are present, I just changed the order for better readability
If I understand correctly this change has no consequences

Reply:
yes, Sorry for that I have not seen